src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -508,12 +574,25 @@ class ContentExplorer extends Component {
                 >
                     {headerActionsAccessory}
                 </ContentExplorerHeaderActions>
-                {isSelectAllAllowed && (
-                    <ContentExplorerSelectAll
-                        numTotalItems={numTotalItems}
-                        isSelectAllChecked={isSelectAllChecked}
+                {isSelectAllAllowed && canIncludeSubfolders ? (
```

Comment:
we might be able to replace `canIncludeSubfolders` with `onIncludeSubfoldersToggle` so we're only passing one prop all the way down instead of two.

i.e. instead of passing two props (one to display and one callback), we can pass just the callback and use that to hide or show the toggle

Reply:
Alright I did this instead. I am going to have to make a small chnge to the other PR as well since that uses canIncludeSubfodlers still